diff --git a/node_modules/@syncfusion/ej2-filemanager/src/file-manager/common/operations.js b/node_modules/@syncfusion/ej2-filemanager/src/file-manager/common/operations.js
index d1f7aa1..f89c720 100644
--- a/node_modules/@syncfusion/ej2-filemanager/src/file-manager/common/operations.js
+++ b/node_modules/@syncfusion/ej2-filemanager/src/file-manager/common/operations.js
@@ -748,7 +748,7 @@ function performReadOperation(parent, result, fn, data, event, operation, target
                 parent.hasId = !isNOU(getValue('id', root));
             }
             if (event === 'finalize-end') {
-                generatePath(parent);
+                // generatePath(parent); // [FIXME INVINCE]: if path set, then wrong path is calculated
             }
         }
     }
